extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/VertxConfiguration.java

```diff
@@ -32,12 +32,20 @@
     /**
      * The maximum amount of time the event loop can be blocked.
      * Default is 2s.
+     * <p>
+     * The value is parsed via the {@link Duration#parse()} method.
+     * If the value consists only of digits the converter treats the value as seconds.
+     * If the value starts with digits the {@code PT} prefix is implicitly prepended to the value.
```

Comment:
There's a note added automatically to the generated documentation with this information, no need to add it here.

Reply:
And maybe I would push these default values to `defaultValue` instead.

Reply:
Well, in the all-config page this info is at the bottom (read invisible) and the link "on the type" points to the `Duration` javadoc...

Reply:
> And maybe I would push these default values to defaultValue instead.

These are Vertx defaults. I'm not quite sure what our strategy is but it seems quite reasonable to keep the Vertx defaults...

Reply:
I agree we should keep the defaults, at least if the current values are OK for us. Now we have two options: either have them in `defaultValue` and be sure the doc is consistent or have the doc out of sync when Vert.x defaults are changed.

As for the duration, there is a small (?) that points to the note. If it's not sufficient, let's tweak the doc output. We don't want to copy this text everywhere we have a duration.

Reply:
> have the doc out of sync when Vert.x defaults are changed.

This is probably what happen the last time ;-). None of the solutions is ideal but we should probably use our own defaults for now.

> As for the duration, there is a small (?) that points to the note. If it's not sufficient, let's tweak the doc output.

It's not that small but somewhat hidden by the javadoc link (one would not expect two links side by side...).

Reply:
> (one would not expect two links side by side...).

I don't see why not? Maybe we should add a tooltip on the (?) icon? Or maybe we should just drop the Javadoc link as I don't think it brings anything useful.

Reply:
> I don't see why not?

Well, usually if there is a link followed by an icon it represents a "single item". It's like a menu item which is usually text + icon.

>  Or maybe we should just drop the Javadoc link 

+1